core/blockchain.go

```diff
@@ -1441,7 +1449,29 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 	}
 	// If we're running an archive node, always flush
```

Comment:
Probably good to update comment to reflect changed code

Reply:
I've added a comment, I am open for any suggestions for improvement